Formula/imageworsener.rb

```diff
@@ -1,8 +1,27 @@
 class Imageworsener < Formula
   desc "Utility and library for image scaling and processing"
   homepage "http://entropymine.com/imageworsener/"
-  url "http://entropymine.com/imageworsener/imageworsener-1.3.1.tar.gz"
-  sha256 "beb0c988c02b1d461dccdb3d6c4fc229316a692ea38689874013ba349dff66d1"
+  revision 1
+
+  stable do
+    url "http://entropymine.com/imageworsener/imageworsener-1.3.1.tar.gz"
+    sha256 "beb0c988c02b1d461dccdb3d6c4fc229316a692ea38689874013ba349dff66d1"
+
+    # Patches to fix a collection of bugs & vulnerabilities.
```

Comment:
Yep. The idea is to rely on the messaging from https://github.com/Homebrew/brew/pull/1732.